setup.py

```diff
@@ -27,34 +27,55 @@
 import os
 import sys
 
-from setuptools import setup
+from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
-requirements = [
-    'alembic>=0.7,<0.8',
-    'Flask>=0.10.1',
-    'six>=1.7.2',
-    'invenio-base>=0.3.0',
-    'invenio-ext>=0.2.1',
-    'invenio-utils>=0.1.1',
-]
 
-test_requirements = [
-    'unittest2>=1.1.0',
+tests_require = [
+    'check-manifest>=0.25',
     'Flask_Testing>=0.4.1',
-    'pytest>=2.8.0',
-    'pytest-cov>=2.1.0',
+    'coverage>=4.0',
+    'isort>=4.2.2',
+    'pep257>=0.7.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
-    'coverage>=4.0.0',
-    'invenio-testing>=0.1.1',
+    'pytest>=2.8.0',
+    'unittest2>=1.1.0',
 ]
 
+extras_require = {
+    'docs': [
+        "Sphinx>=1.3",
+    ],
+    'tests': tests_require,
+}
+
+extras_require['all'] = []
+for reqs in extras_require.values():
+    extras_require['all'].extend(reqs)
 
-class PyTest(TestCommand):
+setup_requires = [
+    'Babel>=1.3',
+]
 
+install_requires = [
+    'Flask-BabelEx>=0.9.2',
+    'Flask-cli>=0.2.1',
+    'Flask>=0.10.1',
+    'alembic>=0.7,<0.8',
```

Comment:
I think it was like this because some upgrades wasn't working with anything bigger than `0.8`, but now that we don't have any upgrade yet probably we can put `>=0.8.3`